configfs/fakertmr/fakertmr.go

```diff
@@ -65,6 +66,8 @@ type RtmrSubsystem struct {
 	RtmrMaps map[int]*rtmrValue
 	// Entries is a map of rtmr entry name to rtmr entry.
 	Entries map[string]*rtmrEntry
+	// If set true, ReadDir will return a list of directory entries.
```

Comment:
I don’t understand how “SetEntries” should be interpreted. Please explain what failure condition you’re trying to make happen. It seems like this setting breaks the fake’s functionality entirely as a simulator for an rtmr subsystem and acts more like a special purpose mock.

if setentries is false, “always return ErrNotExist”. If true, then populate a random directory with some files and return their entries. This doesn’t behave as you would expect if you try to extend two different rtmrs in sequence.